iqrm/core.py

```diff
@@ -54,6 +54,8 @@ def iqrm_mask(x, radius=5, threshold=3.0):
         is 10% of the number of frequency channels
     threshold : float, optional
         Flagging threshold in number of Gaussian sigmas
+    ignorechans: list or ndarray
+        Channels which are known to be "bad" already. Their votes will be ignored
```

Comment:
Their votes aren't ignored in the implementation, I would just say "Channels which are known to be bad already"